README.md

```diff
@@ -1,23 +1,41 @@
-# RAPIDS Documentation Sphinx PyData Theme Assets
+# RAPIDS Sphinx Theme
+
 > **Warning**
-> Work In Progress & Experimental 
+> Work In Progress & Experimental
 
-Using default [PyData Theme](https://pydata-sphinx-theme.readthedocs.io/en/stable/) with customized configuration to standardized RAPIDS documentation navigation. 
+Using default [PyData Theme](https://pydata-sphinx-theme.readthedocs.io/en/stable/) with customized configuration to standardized RAPIDS documentation navigation.
 
 - RAPIDS nav section based on [announcements banner](https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/announcements.html)
 - Custom colors based on [css styling](https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/styling.html)
 - Doc versions based on [version switcher](https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/version-dropdown.html)
 
-
 ## How to Use
-Append the code in the 'conf.py' file into your projects Sphinx config file. Verify that you have up to date 'project' and 'version' variables and no other configurations that would override the options. 
 
+Install the theme:
+
+```sh
+pip install git+https://github.com/rapidsai/sphinx-theme.git@main
+```
+
+Setup the `conf.py` file:
+
+```py
+html_theme = "rapidsai_sphinx_theme"
+```
 
-## Navigation Standardization 
```

Comment:
I'll move this type of content into another readme

Reply:
Sounds good. These links are hardcoded into the theme in `rapidsai_sphinx_theme/__init__.py`, therefore I didn't think this content was necessary.